packages/cardpay-sdk/.eslintrc.cjs

```diff
@@ -4,4 +4,12 @@ module.exports = {
   parserOptions: {
     tsconfigRootDir: __dirname,
   },
+  overrides: [
+    {
+      files: ['*.ts'],
+      rules: {
+        '@typescript-eslint/consistent-type-exports': 'error',
+      },
+    },
+  ],
```

Comment:
Am not sure about using this rule vs ignoring warnings. If we choose this fix, should this be a repo-wide change?

Reply:
This rule makes sure that type exports are specified using the type-only syntax, introduced in Typescript 3.8.

Reply:
I don't have any strong feelings on this change. Maybe @ef4 would like to weigh in

Reply:
> I think it's a good rule, we could upstream it into the shared config too.

Cool, will follow up with a PR for that. Thanks!